pkg/gobmpsrv/gobmpsrv.go

```diff
@@ -117,7 +119,7 @@ func (srv *bmpServer) bmpWorker(client net.Conn) {
 }
 
 // NewBMPServer instantiates a new instance of BMP Server
-func NewBMPServer(sPort, dPort int, intercept bool, p pub.Publisher, splitAF bool) (BMPServer, error) {
+func NewBMPServer(sPort, dPort int, intercept bool, p pub.Publisher, splitAF bool, bmpRaw bool, adminId string) (BMPServer, error) {
```

Comment:
same comment as above, one publisher specific parameters should not change this function signature. Try to pass this information via publisher interface.